setting default pool with a partition + directory definition errors out bigip_ltm_virtual_server

<!--
When filing an issue please check to see if an issue already exists that matches your's issue.

Please open a case (https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

-->

### Environment

 * TMOS/Bigip Version: 16.1.3.1
 * Terraform Version: 1.1.9
 * Terraform bigip provider Version: 1.15.1

### Summary

When referencing a default pool for bigip_ltm_virtual_server that has been created with bigip_ltm_pool as `partition + directory + name` the validation on the pool value fails saying that it must match a specific format:

```
Error: "pool" must match /Partition/Name and contain letters, numbers or [._-:]. e.g. /Common/my-pool
```

### Steps To Reproduce

Steps to reproduce the behavior:

1. Provide terraform resource config which you are facing trouble along with the output of it.
```
resource "bigip_ltm_pool" "test" {
  name                   = "/Common/test-active/test-active_pool_0"
  minimum_active_members = 1
  monitors               = ["/Common/tcp"]
}

resource "bigip_ltm_virtual_server" "test" {
  name        = "/Common/test-active/test-active_vs"
  destination = "10.0.0.10"
  port        = 443
  pool        = bigip_ltm_pool.test.name
  client_profiles = [
    "/Common/tcp-wan-optimized",
    "/Common/test_certificate_profile"
  ]
  server_profiles = [
    "/Common/tcp-lan-optimized"
  ]
}
```

2. To get to know more about the issue, provide terraform debug logs
```
2022-09-22T12:55:53.280-0500 [ERROR] vertex "bigip_ltm_virtual_server.test" error: "pool" must match /Partition/Name and contain letters, numbers or [._-:]. e.g. /Common/my-pool
2022-09-22T12:55:53.280-0500 [ERROR] vertex "bigip_ltm_virtual_server.test" error: "pool" must match /Partition/Name and contain letters, numbers or [._-:]. e.g. /Common/my-pool
2022-09-22T12:55:53.281-0500 [ERROR] vertex "bigip_ltm_virtual_server.test(expand)" error: "pool" must match /Partition/Name and contain letters, numbers or [._-:]. e.g. /Common/my-pool
```

3. To capture debug logs, export TF_LOG variable with debug ( export TF_LOG= DEBUG ) before 
  runnning terraform apply/plan

4. As3/DO json along with the resource config( for AS3/DO resource issues )


### Expected Behavior
When you create pools with `partition + directory + name` formatting and reference them in bigip_ltm_virtual_server as the default pool it should be allowed.

### Actual Behavior
When referencing a pool created as `partition + directory + name` it fails because the pool does not follow the simple formatting of /Common/$pool

--- a/bigip/resource_bigip_ltm_virtual_server.go
+++ b/bigip/resource_bigip_ltm_virtual_server.go
@@ -73,7 +73,7 @@ func resourceBigipLtmVirtualServer() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Description:  "Default pool for this virtual server",
-				ValidateFunc: validateF5Name,
+				ValidateFunc: validateF5NameWithDirectory,
 			},
 
 			"mask": {

